packages/vue/config/storybook/config.js

```diff
@@ -8,13 +8,13 @@ import "../../src/css/_all.scss";
 
 const req = require.context("../../src/", true, /.stories.js$/);
 
-function PaddingDecorator() {
-  return {
-    template: '<div style="padding: 20px"><story/></div>'
-  };
-}
+// function PaddingDecorator() {
```

Comment:
please remove it completely if it's not in use.

Reply:
https://github.com/DivanteLtd/storefront-ui/pull/492/commits/ee9b7dddad2bb90a1f59a2827b89ad74317377d5